test/fixtures/export/Do not caps-protect name fields #384.noopsort.bibtex

```diff
@@ -179,17 +179,17 @@
 }
 
 @book{este,
-  author = {\noopsort{Este}d'\relax{} Este, Beatrice},
+  author = {{\noopsort{este}}d'\relax Este, Beatrice},
   note = {[Este][Beatrice d']}
 }
 
 @book{medici,
-  author = {\noopsort{Medici}de'\mbox{} Medici, Lorenzo},
-  note = {[Medici][Lorenzo de'\mbox]}
+  author = {{\noopsort{medici}}de' Medici, Lorenzo},
+  note = {[Medici][Lorenzo de'U+200B]}
 }
 
 @book{al-hakim,
-  author = {\noopsort{Hakim}{al-\relax{} Hakim}, Tawfiq},
+  author = {{\noopsort{hakim}}{al-\relax Hakim}, Tawfiq},
```

Comment:
I thought you said the `\relax{}<space>` format was OK? It's much harder for me to generate `\relax<space>` here because the general algorithm I use prefers to write out `\<command>{}` over `\<command><space>` because if the user does have a space after a character that generates a LaTeX command, the `{}` makes sure I respect that space. Counteracting for that would complicate the algorithm.